docs/language/access-control.md

```diff
@@ -99,7 +99,7 @@ To summarize the behavior for functions:
 | `access(account)`        | Current, inner, and other contracts in same account |
 | `pub` / `access(all)`    | **All**                                             |
 
-Declarations of structures, resources, events, and [contracts](../contracts) can only be public.
+Declarations of structures, resources, events, and [contracts](https://github.com/onflow/cadence/blob/master/docs/language/contracts.md) can only be public.
```

Comment:
The relative links are correct here, the links should not link to the source files: The documentation source files in this repository are rendered on https://docs.onflow.org/cadence/language.

Could you please revert this and the other similar link changes below?

Reply:
Actually, all the relative links that I have updated are not working properly, which move to that link it showing you "404, page not found" error, therefore I used absolute path from the source.
Let me revert all these links according to https://docs.onflow.org/cadence/language

Reply:
Now I get that, the path in the github repository is fine but when you move to https://docs.onflow.org/cadence/language you can see both relative links at the end of the page or not working 

> Disclaimer: In real Cadence code, all type definitions and code must be declared and contained in [contracts](https://docs.onflow.org/cadence/contracts/) or [transactions](https://docs.onflow.org/cadence/transactions/), but we omit these containers in examples for simplicity.

let me fix these links

Reply:
@ObjectPlayer we should hold on converting to links pointing to flow docs site, as the relative linking issue has been addressed by PR https://github.com/onflow/flow/pull/764/

Reply:
@kerrywei fixed the relative links in #1372, I updated this PR (merged master). Relative links will also help us build and test the documentation locally. I left the other two improvements in the PR, as they are good fixes. Thank you again for raising this issue and helping to fix it @ObjectPlayer 🙏